### PR TITLE
Move AudioAtom to DCR

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Move AudioAtom from Atoms-Rendering in CSNX into the DCR repo. This is a direct port, with a number of very minor linting fixes.

## Why?

It's only used in DCR, therefore there's no reason for it to exist elsewhere.

### CODE REVIEW

File: dotcom-rendering/src/components/AudioAtom/AudioAtom.tsx [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+					preload="none"
+					data-component="inarticle audio"
+					data-duration={durationTime}
+					data-media-id={id ?? '_no_ids'}
+					data-title={titleStyle}
+				>
+					<p>
+						Sorry your browser does not support audio - but you can
+						download here and listen
+						https://audio.guim.co.uk/2020/05/05-61553-gnl.fw.200505.jf.ch7DW.mp3
```
Comment (georgeblahblah) [inline dotcom-rendering/src/components/AudioAtom/AudioAtom.tsx:346]: I'm curious why this is hardcoded - I wonder if we should be using `urlToUse` instead?

```suggestion
						${urlTouse}
```

File: dotcom-rendering/src/lib/decideAppsRendering.ts
Comment (georgeblahblah) [inline dotcom-rendering/src/lib/decideAppsRendering.ts:1]: Do you think we could use [`pillars.ts`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/pillars.ts) instead, and delete this module?

**Review by georgeblahblah — COMMENTED** [review]
Thanks @joecowton1! I had one question about the `decideAppsRendering.ts` module and a hardcoded audio URL.

There's a few other things that we might be able to tidy up to adopt the DCR style, but we it's probably better to address those in a subsequent PR

**Review by JamieB-gu — COMMENTED** [review]
Looks great, thanks Joe! 👍 (happy to approve, but will wait for @georgeblahblah 's comments)

File: dotcom-rendering/src/lib/decideAppsRendering.ts
Comment (joecowton1) [reply dotcom-rendering/src/lib/decideAppsRendering.ts:1]: Used decidePalette, has involved a bit of refactor but feels more idiomatic to DCR now.

**Review by georgeblahblah — APPROVED** [review]

**Pull request merged**